R/knownVariables.r

```diff
@@ -30,6 +30,15 @@ four_digit_identifiers <- c("Calendar year", "Reporting year", weeks, months)
 
 six_digit_identifiers <- c("Autumn and spring term", terms, "Financial year", "Academic year", "Tax year", financial_quarters, financial_halves)
 
+# Latest GSS/ONS data symbols
+gssNAvcode <- "x"
+gssNApcode <- "z"
+gssSupcode <- "c"
+gssRndcode <- "low"
+Symbol <- (c(gssNApcode, gssNAvcode, gssSupcode, gssRndcode))
```

Comment:
`Symbol` should be more informative so it makes more sense in the other code scripts, e.g. `gss_symbols`

Reply:
Also - is there a reason for having the extra brackets around the c() like (c()) ?